lib/authlogic/session/password.rb

```diff
@@ -258,17 +266,6 @@ def password_field
           def verify_password_method
             self.class.verify_password_method
           end
-
-          # In Rails 5 the ActionController::Parameters no longer inherits from
-          # HashWithIndifferentAccess. (http://bit.ly/2gnK08F) This method
-          # converts the ActionController::Parameters to a Hash.
-          def parse_param_val(value)
-            if value.first.class.name == "ActionController::Parameters"
-              [value.first.to_h]
```

Comment:
I'm confused why this was working for everyone before -- this is before wrapping in an array, so it should be `value.class.name` instead of `value.first.class.name`.